rate_limiter.rst

```diff
@@ -353,7 +353,7 @@ the :class:`Symfony\\Component\\RateLimiter\\Reservation` object returned by the
             $limit = $limiter->consume();
             $headers = [
                 'X-RateLimit-Remaining' => $limit->getRemainingTokens(),
-                'X-RateLimit-Retry-After' => $limit->calculateTimeForTokens(1, 1),
+                'X-RateLimit-Retry-After' => $limit->getRetryAfter()->getTimestamp(),
```

Comment:
The header must contain the number of seconds from now to retry I believe. So we can do something like:

```suggestion
                'X-RateLimit-Retry-After' => now() - $limit->getRetryAfter()->getTimestamp(),
```

And then add `use function Symfony\Component\Clock\now;` at the top of the example.